src/Bicep.Core.UnitTests/Utils/ResultTests.cs

```diff
@@ -32,7 +32,7 @@ public void Result_should_encapsulate_error_result()
         test.IsSuccess().Should().BeFalse();
         test.TryUnwrap().Should().BeNull();
         FluentActions.Invoking(() => test.Unwrap()).Should().Throw<InvalidOperationException>()
-            .Which.Message.Should().Be("Cannot unwrap a failed result.");
+            .Which.Message.Should().Be("Cannot unwrap a failed result: error!.");
```

Comment:
?  

Also, ends with "!."

Reply:
@anthony-c-martin ?

Reply:
Sorry, forgot to respond. Yes this was intentional - I just wanted to log the contents of the error if there is one in `Unwrap()`, since it's helpful for debugging. The `!` comes from the mock data on line 30.